docs/prover/approx/harnessing.md

```diff
@@ -1,31 +1,52 @@
-Harnessing
-==========
+# Harnessing
 
 Occasionally, CVL lacks a feature that is necessary for a complete verification
 of a contract.  We are working to extend the feature set of CVL to cover these
 cases, but in the mean time we have developed a set of workarounds that we
 refer to as "harnesses".
 
-Harnesses involve systematically altering the behavior of the code being
-verified.  They are therefore {term}`unsound`.
-
-
-Harnessing by extension
------------------------
-
-```{todo}
-This technique is currently undocumented.  See our [OpenZeppelin verification
-project](https://github.com/OpenZeppelin/openzeppelin-contracts/tree/master/certora) for an example.
+## Example: CometHarnessWrappers
+
+Consider a scenario where we want to write a unit test for an internal functions of a contract. The CometHarnessWrappers contract serves as a workaround, allowing us to call original functions rather than relying on summarized implementations. 
+
+
+```solidity
+// SPDX-License-Identifier: XXX ADD VALID LICENSE
+pragma solidity ^0.8.11;
+
+import "./CometHarnessGetters.sol";
+import "../munged/ERC20.sol";
+import "../munged/vendor/@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
```

Comment:
instead of all the imports, I'll put a stub for the contract we're harnessing, to show and not just explain that we expose internal functions or allow new functionality (as is explained later). otherwise it's not very clear

Reply:
Added link to the source and dropped imports